services/arv-git-httpd/git_handler_test.go

```diff
@@ -41,13 +41,14 @@ func (s *GitHandlerSuite) TestEnvVars(c *check.C) {
 }
 
 func (s *GitHandlerSuite) TestCGIError(c *check.C) {
+	log.Printf("git_handler_test: TestCGIError() - expecting a SplitHostPort error, so it is safe to ignore!")
```

Comment:
That's fair—but the reason we write out a special message in that case is because we have no control over the installer code.  I think our own test code follows the pattern Tom describes.